package.json

```diff
@@ -1,80 +1,53 @@
 {
-  "name": "react-scroll-sync",
-  "version": "0.0.0-development",
-  "description": "Synchronize scroll positions across multiple scrollable containers",
-  "main": "dist/index.js",
-  "scripts": {
-    "clean": "rimraf ./dist",
-    "prebuild": "npm run lint",
-    "build": "NODE_ENV=production npm run clean && webpack",
-    "prepare": "npm run build",
-    "lint": "eslint src --ext .js,.jsx",
-    "lint:fix": "npm run lint -- --fix",
-    "lint:staged": "lint-staged",
-    "styleguide:server": "styleguidist server",
-    "styleguide:build": "styleguidist build"
-  },
-  "lint-staged": {
-    "*.js": [
-      "eslint --fix",
-      "git add"
-    ]
-  },
-  "pre-commit": "lint:staged",
```

Comment:
Not a fan seeing this is gone. Let's add it back. Feel free to use the latest versions and follow the install guide.

Reply:
Done